commonmark-ext-autolink/src/main/java/org/commonmark/ext/autolink/internal/AutolinkPostProcessor.java

```diff
@@ -11,9 +12,40 @@
 
 public class AutolinkPostProcessor implements PostProcessor {
 
-    private LinkExtractor linkExtractor = LinkExtractor.builder()
-            .linkTypes(EnumSet.of(LinkType.URL, LinkType.EMAIL))
-            .build();
+    private final LinkExtractor linkExtractor;
+
+    public AutolinkPostProcessor() {
+        this(EnumSet.of(AutolinkType.URL, AutolinkType.EMAIL));
+    }
```

Comment:
This constructor shouldn't be necessary, as this is an internal class (in `internal` package).